### PR TITLE
Update path for policyFile

### PR DESCRIPTION
**Pull request by jboxman**
- https://bugzilla.redhat.com/show_bug.cgi?id=1684482

### CODE REVIEW

**Comment by jboxman** [comment]
QE verified in BZ[0].

@openshift/team-documentation - PTAL, thanks!

[0] https://bugzilla.redhat.com/show_bug.cgi?id=1684482

File: install_config/master_node_configuration.adoc [hunk 1]
```diff
@@ -10,6 +10,8 @@
 
 toc::[]
 
+== Overview
```
Comment (kalexand-rh) [inline install_config/master_node_configuration.adoc]: Use a more descriptive heading, not "Overview."

File: install_config/master_node_configuration.adoc [hunk 2]
```diff
@@ -92,7 +94,7 @@ inventory file with some substituted values. Modifying this inventory file and r
 
 While {product-title} supports using Ansible for cluster installation, using an Ansible playbook and inventory file, you can also use other management tools, such as
 https://puppet.com/[Puppet], https://www.chef.io/[Chef],
-http://saltstack.com/[Salt]).
+http://saltstack.com/[Salt].
```
Comment (kalexand-rh) [inline install_config/master_node_configuration.adoc]: add "or"

**Review by kalexand-rh — COMMENTED** [review]
Couple of picks. Overall, LGTM

**Pull request merged**